ictst/texts/data-dictionary.md

```diff
@@ -1,5 +1,252 @@
+* Infra / Datacenter
+* Infra / Cloud (for example: GCP, AWS, Azure, OVH Cloud)
+* Infra / Network
+* Equipment / End-user Devices
+* Equipment / IoT
+* Equipment / Industrial equipment
+* Software / Development Tools
+* Software / CI/CD
+* Software / Backend
+* Software / API
+* Software / Mobile Apps (for example: Android App, iPhone App)
+* Software / Web Apps (for example: React, Spring, Node.js, ROR, PHP apps)
+* Software / Web Content (for example: WordPress, Drupal, Prestashop, Magento, etc.)
+* TBQ: To be Qualified
+
+## Assessed Functional Unit ([assessed_functional_unit])**✅**
+
+Which granularity of a digital service or an information system can be assessed with the tool (it does not include the data used as entry point which are covered in the field [data_entry_point]):
+
+* Process (System)
+* Application
+* Team
+* Project
+* User
+* Request (http)
+* Feature
+* User Story
+* Machine
+* Room
+* Website
+* Network Traffic
+* TBQ: To be Qualified
+
+##  Data Entry point ([data_entry_point])**✅**
+
+List all the data used to produce the environmental indicators.
+
+* Server plan
+* CPU based (RAPL)
+* CPU, RAM, I/O (raw)
+* Bandwidth
+* DOM Size
+* Request Number
+* User Journey Modelisation
+* User localization
+* Datacenter Localization
+* Connexion type
+* PUE
+* WUE
+* Mutualization rate
+* Devices Inventory (both internal and external users of an application)
+* CPU capacity
+* RAM capacity
+* I/O Capacity
+* Traffic/Usage
+* Storage
+* Undisclosed
+* TBQ: To be Qualified
+
+## Life Cycle Steps ([life_cycle_steps])**✅**
+
+Qualify which steps of a Life Cycle Assessment are tracked. Minimum coverage is usually “usage” when only the energy consumed during the use phase is taken into consideration. This field enables also to qualify the scope(s) being covered for the GWP indicator (GHG emissions in CO2e). Usual steps of an [ISO 14040:2006 LCA](https://www.iso.org/standard/37456.html) are eligible:
+
+* Extraction
+* Manufacturing
+* Transport
+* Usage
+* End of life
+* Not Applicable
+* Undisclosed
+* TBQ: To be Qualified
+
+## GHG scope covered ([ghg_scope])**✅**
+
+This category makes it easy to link the product view (life cycle steps) with the Corporate Standard (company scopes), for educational purposes.
+
+If GWP is checked in the environmental indicator field, this field indicates which scopes are covered (following [GHG protocol](https://ghgprotocol.org/) Corporate Standard):
+
+* Scope 1 and 2 (market-based)
+* Scope 1 and 2 (location-based)
+* Scope 1, 2 (market-based) and 3
+* Scope 1, 2 (location-based) and 3
+
+Should be aligned with the Life Cycle Steps (no scope 3 can be indicated if Manufacturing is not included in the Life Cycle Steps).
+
+![The relationship between the Corporate, Scope 3, and Product Standards for a company manufacturing product A (Extracted from GHG Protocol Product Standard)](/api/attachments.redirect?id=8464dcbc-6acc-44e9-8a32-d40b0b0ef4a2" =368x174")
```

Comment:
This URL seems broken and I'm not sure where it should point to.

Reply:
An image was missing, it's now fix.